src/components/SigningOfficialDaaAgreementWrapper.js

```diff
@@ -93,8 +98,8 @@ export const SigningOfficialDaaAgreementWrapper = (props) => {
 // Wraps component and ensures that SO agrees to the
 // Broad and NIH agreements before proceeding to the given
 // component.
-export const ensureSoHasDaaAcknowledgement = (component) => {
-  return (props) => h(SigningOfficialDaaAgreementWrapper, {}, [
+export const ensureSoHasDaaAcknowledgement = (component, isDataSubmitter=false) => {
```

Comment:
```suggestion
export const ensureSoHasDaaAcknowledgement = (component, isDataSubmitterTab=false) => {
```

One minor suggestion, switch `isDataSubmitter` to `isDataSubmitterTab` across the board since the user role isn't what is actually changing.